bundles/org.openhab.binding.roku/src/main/resources/OH-INF/thing/roku.xml

```diff
@@ -53,6 +55,8 @@
 			<channel id="programTitle" typeId="programTitle"/>
 			<channel id="programDescription" typeId="programDescription"/>
 			<channel id="programRating" typeId="programRating"/>
+			<channel id="power" typeId="system.power"/>
```

Comment:
`power` could be moved to the top since it is the most important control for a TV.

Reply:
Easy enough.